src/components/forms/controls/Textarea/Textarea.module.scss

```diff
@@ -0,0 +1,38 @@
+/* Copyright (c) Fortanix, Inc.
+|* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of
+|* the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@use '../../../../styling/defs.scss' as bk;
+
+@layer baklava.components {
+  .bk-textarea {
+    @include bk.component-base(bk-textarea);
+
+    --bk-textarea-border-color: #{bk.$theme-text-area-border-default};
+
+    border: 1px solid var(--bk-textarea-border-color);
+    border-radius: bk.$radius-1;
+    color: bk.$theme-text-area-text-default;
+    padding: bk.$spacing-3;
+
+    &:focus, &.pseudo-focused {
+      outline: none !important;
```

Comment:
You can use `@include bk.focus-hidden` to hide it instead (note: should be at the top level, not inside a `&:focus`).